pkg/insight/applicationcount.go

```diff
@@ -27,7 +44,86 @@ type ApplicationCountByLabelSet struct {
 
 type ApplicationCountLabelSet struct {
 	// KUBERNETES, TERRAFORM, CLOUDRUN...
-	Kind string `json:"kind"`
+	Kind model.ApplicationKind `json:"kind"`
 	// enable, disable or deleted
-	Status string `json:"status"`
+	Status ApplicationStatus `json:"status"`
+}
+
+func NewApplicationCount() *ApplicationCount {
+	counts := make([]ApplicationCountByLabelSet, len(model.ApplicationKind_name)*len(statuses))
+	for _, k := range model.ApplicationKind_value {
+		for j, s := range statuses {
+			counts[int(k)*len(statuses)+j] = ApplicationCountByLabelSet{
+				LabelSet: ApplicationCountLabelSet{
+					Kind:   model.ApplicationKind(k),
+					Status: s,
+				},
+			}
+		}
+	}
```

Comment:
Maybe we don't need this too. After receiving a list of applications, we calculate the `Counts` and override `Counts` field of this `ApplicationCount`.